src/components/Blocking/ItemList.js

```diff
@@ -30,7 +31,7 @@ export default ItemList;
 const styles = StyleSheet.create({
   container: {
     paddingVertical: 18,
-    borderBottomColor: '#E0E0E0',
+    borderBottomColor: COLORS.alto,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with dynamic colors from a theme object is a good practice. It enhances maintainability and ensures consistency throughout the application.
```diff
- borderBottomColor: '#E0E0E0',
+ borderBottomColor: COLORS.alto,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->